src/components/Outputs.tsx

```diff
@@ -29,20 +28,11 @@ const Outputs: React.FC<OutputProps> = ({ roles, data, generatedToken }) => {
             spellCheck='false'
             readOnly
             value={`
-  {
-      "name": ${data.name ? `"${data.name}",` : ''}
-      "company": ${data.company ? `"${data.company}",` : ''}
-      "roles": ${
-        roles.length > 0
-          ? `"roles": [ ${roles.map(role => {
-              return `
-        "${role.trim()}"`;
-            })}
-      ],`
-          : ''
-      }
-      ${data.aud ? `"aud": "${data.aud}",` : ''}
-  }`}
+        {     ${Object.entries(data).map(key => {
```

Comment:
Good simplification, but I think there will be some problems with the roles array now being handled as objects.